fdbclient/BlobGranuleReader.actor.cpp

```diff
@@ -102,13 +102,13 @@ ACTOR Future<RangeResult> readBlobGranule(BlobGranuleChunkRef chunk,
 		}
 
 		state int numDeltaFiles = chunk.deltaFiles.size();
-		state StringRef* deltaData = new (arena) StringRef[numDeltaFiles];
+		state std::vector<StringRef> deltaData;
```

Comment:
NIt: This technically changes the memory lifetime of this data. Since both the arena and this vector are scoped to the lifetime of this actor it's fine.
But if we wanted to change that, a VectorRef<StringRef> would be preferable to an std::vector<StringRef>

Reply:
Yes, this is noticed and considered. I did hesitate about using `VectorRef` over `std::vector` but decided to use std libraries unless you really wanted it to be VectorRef.